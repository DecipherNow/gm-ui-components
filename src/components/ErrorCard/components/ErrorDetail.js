import styled from "styled-components";
import { keen } from "style/styleVariables";

const ErrorDetail = styled.p`
  align-items: center;
  display: flex;
  flex: 1;
  height: auto;
  justify-content: center;
  font-family: ${props => props.theme.FONT_STACK_BASE};
  font-size: ${props => props.theme.FONT_SIZE_SM};
  margin: 0;
  opacity: ${props => props.theme.OPACITY_70};
`;

ErrorDetail.defaultProps = {
  theme: keen
};

ErrorDetail.displayName = "ErrorDetail";

export default ErrorDetail;
