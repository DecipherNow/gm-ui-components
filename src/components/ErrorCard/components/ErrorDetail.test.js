import React from "react";
import { shallow } from "enzyme";
import ErrorDetail from "./ErrorDetail";

describe("ErrorDetail", () => {
  it("matches snapshot", () => {
    const aErrorDetail = shallow(<ErrorDetail />);
    expect(aErrorDetail).toMatchSnapshot();
  });
});
