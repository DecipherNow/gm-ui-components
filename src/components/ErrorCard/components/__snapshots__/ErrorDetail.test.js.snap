// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorDetail matches snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  height: auto;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
  font-size: 11px;
  margin: 0;
  opacity: 0.7;
}

<p
  className="c0"
/>
`;
